frontend/src/metabase-lib/queries/utils/card.js

```diff
@@ -117,24 +117,24 @@ export function applyParameters(
     const options =
       deriveFieldOperatorFromParameter(parameter)?.optionsDefaults;
 
+    const queryParameter = {
+      type,
+      value: normalizeParameterValue(type, value),
+      id: parameter.id,
+    };
+
+    if (options) {
+      queryParameter.options = options;
```

Comment:
This is the only real change. 

If the options are undefined, don't pass them.